Dockerfile

```diff
@@ -1,5 +1,15 @@
-FROM node:20-alpine
-RUN apk add g++ make python3 dumb-init
+FROM node:20-bullseye
+RUN set -ex \
+    && sed -i -- 's/# deb-src/deb-src/g' /etc/apt/sources.list \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+               build-essential \
+               cdbs \
+               devscripts \
+               equivs \
+               fakeroot \
+    && apt-get clean \
+    && rm -rf /tmp/* /var/tmp/*
```

Comment:
Just grabbed the snippet somewhere, pretty much clueless about Docker scripts. @omBratteng let me know if we should change something about this.

Reply:
I'll hopefully be back tomorrow, so I'll take a look

Reply:
No rush, just tagging you for more visibility 🙏

Reply:
On line 21-23 there's a `RUN` that must be removed